README.md

```diff
@@ -19,111 +19,205 @@
+fx aggregator certify --fqdn $FQDN  # Press "y" when prompted.
+```
+#### _Step 1e: Zip/Export Aggregator Workspace._
+```bash
+fx workspace export # Zip/export Workspace to be able to be imported by collaborators for usage.
+```
+### Step 2: Setup Collaborator1 _(in new terminal of same machine)_. _(Tip:Press Ctrl+Alt+T)_
+_Note: Steps for new terminal in another machines are similar expect for exchanging files(cp vs scp) between these machines_
+#### Prerequisite: _Setup FQDN_
+```bash
+echo $(hostname --all-fqdns |  awk '{print tolower($1)}') # Find  the right FQDN
+```
+```bash
+export FQDN=<FQDN> # Enter your FQDN here
+export no_proxy= localhost,<local machine IP>,<FQDN>
+```
+#### _Step 2a: Setup Collaborator1 Variables._
+```bash
+export COLLAB_NUMBER=1 # need to define the collab number. (1 now, 2 later, etc)
+```
+```bash
+conda activate fedai # If machine is different you need Step 0: Prerequisites.
+export COLLAB_NAME=cob_${COLLAB_NUMBER}
+export COLLAB_PATH=${HOME}/${COLLAB_NAME}
+mkdir ${COLLAB_PATH}
+```
+#### _Step 2b: Setup Collaborator1 Workspace Variables._
+```bash
+export WORKSPACE_NAME="my_federation"
+export WORKSPACE_PATH_AGGR=${HOME}/${WORKSPACE_NAME}
+export WORKSPACE_PATH_COBR=${COLLAB_PATH}/${WORKSPACE_NAME}
+```
+#### _Step 2c: Setup Collaborator1 Workspace._
+```bash
+cp ${WORKSPACE_PATH_AGGR}/${WORKSPACE_NAME}.zip ${COLLAB_PATH}/${WORKSPACE_NAME}.zip
+cd ${COLLAB_PATH}
+fx workspace import --archive ${WORKSPACE_NAME}.zip # Import the copied workspace from aggregator
+```
+#### _Step 2d: Create Collaborator1 Workspace Certificate._
+```bash
+cd ${WORKSPACE_PATH_COBR}
+fx collaborator create -n ${COLLAB_NAME} -d ${COLLAB_NUMBER}
+fx collaborator generate-cert-request -n ${COLLAB_NAME}
+```
+#### _Step 2e: Certify collaborator workspace using a Certificate Authority (The Aggregator also serves as the CA here)_
+```bash
+cp ${WORKSPACE_PATH_COBR}/col_${COLLAB_NAME}_to_agg_cert_request.zip ${WORKSPACE_PATH_AGGR}/  # Send collaborator certificate to aggregator
+```
+**_Switch to Aggregator(CA) Terminal and certify the collaborator certificate and send it back to collaborator_.**
+```bash
+export COLLAB_NUMBER=1 #Specify the collaborator number whose certificate you want to certify
+```
+```bash
+export COLLAB_NAME=cob_${COLLAB_NUMBER}
+export COLLAB_PATH=${HOME}/${COLLAB_NAME}
+export WORKSPACE_PATH_COBR=${COLLAB_PATH}/${WORKSPACE_NAME}
+cd ${WORKSPACE_PATH_AGGR}
+fx collaborator certify --request-pkg col_${COLLAB_NAME}_to_agg_cert_request.zip  # Certify the collaborator certificate. Press "y" when asked.
+```
+```bash
+cp ${WORKSPACE_PATH_AGGR}/agg_to_col_${COLLAB_NAME}_signed_cert.zip ${WORKSPACE_PATH_COBR}/ # Send signed certificate back to collaborator
+```
+**_Return back to Collaborator Terminal and import the signed certificate sent from CA(Aggregator)._**
+```bash
+fx collaborator certify --import agg_to_col_${COLLAB_NAME}_signed_cert.zip #import signed certificate rfom CA.
```

Comment:
Here I have a error of "Error: Invalid value", because the Collaborator Terminal don't know the name of "COLLAB_NAME" (cob_1, for the file: agg_to_col_cob_1_signed_cert.zip ), probably need to clarify or update the command